### ISSUE TITLE
Allow selecting gift characters on new character selection after completing the gift character's adventure

### ISSUE THREAD

**Issue by cfogrady** [cfogrady/VitalWear#56]

**Comment by Th3Alic3** [comment]
I attempted to use the BE Imperialdramon Dim and notice Adventure Level 2 did not add a new Character of Leafmon to my available Characters.

So, I wanted to clarify: this Issue (#56) is here because the BE Adventures are not currently programmed to reward a Gift at all OR is me not receiving the Leafmon a glitch?

**Comment by cfogrady** [comment]
Correct. At the moment gift characters simply aren't implemented. This is higher on my priority list. It will probably be included in the version after 0.5.0
